core/src/main/java/org/kohsuke/stapler/HttpResponses.java

```diff
@@ -37,6 +39,12 @@
  * @author Kohsuke Kawaguchi
  */
 public class HttpResponses {
+
+    public interface ErrorCustomizer {
+        @CheckForNull
```

Comment:
Nit: being a new API, it might be preferable to use `Optional<HttpResponseException>` instead, though thanks to the pervasive tooling, the annotation works almost the same, so up to you.

Reply:
Using new code style / pattern on an existing code base makes the refactoring harder and the new stuff / trend will change every x years. I am against such changes that are reducing the consistency of the code style of the project. As you say, there is no added value.

Reply:
I'm ambivalent about it, so I'll just leave it as is.